WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -187,9 +187,25 @@ def _processResourceDescription( self, resourceDescription ):
         if resourceDescription.has_key( name ):
           resourceDict[name] = resourceDescription[name]
 
-      if resourceDescription.has_key( 'JobID' ):
+      if 'JobID' in resourceDescription:
         resourceDict['JobID'] = resourceDescription['JobID']
 
+      # Convert MaxRAM and NumberOfCores parameters into a list of tags
+      maxRAM = resourceDescription.get( 'MaxRAM' )
+      nCores = resourceDescription.get( 'NumberOfCores' )
```

Comment:
These are different things. I have overlooked it in the #2673. These parameters are added as Tags to the resource description in #2673. This is not needed any more as the tags generation is in the Matcher now. Rather add MaxRAM parameter ( in MBs ) as in the CS now. The same for Processors_N. Add NumberOfProcessors ( Oh ) int parameter.